google/cloud/bigquery/v2/minimal/internal/job_request.cc

```diff
@@ -163,12 +163,20 @@ StatusOr<rest_internal::RestRequest> BuildRestRequest(
     request.AddQueryParameter("maxResults", std::to_string(r.max_results()));
   }
   if (r.min_creation_time()) {
-    request.AddQueryParameter("minCreationTime",
```

Comment:
Can you detail what value the BQ API is returning so I can understand why this changes was needed? thanks.

Reply:
Docs states that min/maxCreationTime is a string with `value for job creation time, in milliseconds since the POSIX epoch`
https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs/list#query-parameters
Currently the format is like 2023-11-03T10:33:16.946721928Z which is not valid for BQ API.

https://screenshot.googleplex.com/7UZJdsdFL7KXUX8

Reply:
Got it. Thanks.